navigator/frontend/tslint.json

```diff
@@ -50,7 +50,12 @@
     "object-literal-key-quotes": [true, "consistent-as-needed"],
     "object-literal-shorthand": true,
     "one-variable-per-declaration": [true],
-    "ordered-imports": [true],
+    "ordered-imports": [
+      true,
+      {
+        "named-imports-order": "case-insensitive-legacy"
```

Comment:
This was what we had before as well. The defaults just changed and given the switch to eslint, it didn’t seem worth trying to make the change now.